package.json

```diff
@@ -49,5 +50,13 @@
     "singleQuote": true,
     "trailingComma": "all",
     "arrowParens": "always"
+  },
+  "lint-staged": {
+    "*.rs": "rustfmt --edition 2021",
```

Comment:
We don't need format rs, since we have https://github.com/shulaoda/rolldown/blob/f8aca97e0fe01011524d34e3c82bba668ef0ecd4/.vscode/settings.json#L2-L5

Reply:
You are right.